### PR TITLE
HelicsBool handling and comments modification

### PR DESCRIPTION
**Pull request by afisher1**
first attempt at adding % in front of all function comments to make use of help feature in MATLAB.
also modified HelicsBool arguments to accept
logcials or int/float values of 0 and 1.

### CODE REVIEW

File: matlabBindings/+helics/helicsBrokerSetLogFile.m [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-	Set the log file on a broker.
+%	Set the log file on a broker.
 
-	@param broker The broker to set the log file for.
-	@param logFileName The name of the file to log to.
+%	@param broker The broker to set the log file for.
+%	@param logFileName The name of the file to log to.
 
-	@param[in,out] err An error object that will contain an error code and string if any error occurred during the execution of the function.
+%	@param[in,out] err An error object that will contain an error code and string if any error occurred during the execution of the function.
```
Comment (phlptp) [inline matlabBindings/+helics/helicsBrokerSetLogFile.m:7]: Don't need to do this now but we might need to have some sort of script that goes through and cleans these up.  For example the err param is not used in matlab

**Comment by phlptp** [comment]
@afisher1  is this change included in the 3.3 stuff you did or is this separate?

**Comment by nightlark** [comment]
It looks like the changes here were not included in PR #10

**Comment by afisher1** [comment]
reading the MATLAB documentation the help text needs to be just below the function definition in the .m files so I'll need to change that.

**Comment by afisher1** [comment]
I made the requested changes to the function help comments. removed the multi comment brackets and removed all occurences of param[in,out] err lines from the generated functions.

**Review by phlptp — APPROVED** [review]

**Pull request merged**